deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -2481,6 +2491,29 @@ NAN_METHOD(WebGLRenderingContext::SetTopLevel) {
   gl->topLevel = topLevel;
 }
 
+NAN_METHOD(WebGLRenderingContext::FramebufferTextureMultiviewOVR) {
+  GLenum target = TO_UINT32(info[0]);
+  GLenum attachment = TO_UINT32(info[1]);
+  WebGLTexture? texture = TO_INT32(info[2]);
+  GLint level = TO_INT32(info[3]);
+  GLint baseViewIndex = TO_INT32(info[4]);
+  GLsizei numViews = TO_UINT32(info[5]);
+
+  glframebufferTextureMultiviewOVR(target, attachment, texture, level, baseViewIndex, numViews);
+}
+
+NAN_METHOD(WebGLRenderingContext::FramebufferTextureMultisampleMultiviewOVR) {
+  GLenum target = TO_UINT32(info[0]);
+  GLenum attachment = TO_UINT32(info[1]);
+  GLuint texture = TO_INT32(info[2]);
```

Comment:
This one will need to be extracted out of `.id`.

Our implementation of `WebGLTexture` is actually `{id: GLID}` (in most implementations it is completely opaque). But it is not a `GLuint`.

Reply:
I made it as `GLuint` based off [here](https://github.com/KhronosGroup/OpenGL-Registry/blob/master/extensions/OVR/OVR_multiview_multisampled_render_to_texture.txt#L53).

Reply:
It's correct that it's a `GLuint` at the OpenGL level. But WebGL does not use ints for textures. It uses `WebGLTexture`, which contains the int in an abstract way. In our case that's concretely `tex.id`.

Reply:
8ece516